backend/database/migrations/create-users.js

```diff
@@ -1,36 +1,72 @@
-'use strict';
 module.exports = {
-  up: (queryInterface, Sequelize) => {
-    return queryInterface.createTable('users', {
-      id: {
-        allowNull: false,
-        autoIncrement: true,
-        primaryKey: true,
-        type: Sequelize.INTEGER
-      },
-      username: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      email: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      password: {
-        allowNull: false,
-        type: Sequelize.STRING
-      },
-      createdAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      },
-      updatedAt: {
-        allowNull: false,
-        type: Sequelize.DATE
-      }
-    });
-  },
-  down: (queryInterface, Sequelize) => {
-    return queryInterface.dropTable('users');
-  }
-};
+    up: (queryInterface, Sequelize) =>
+        queryInterface.createTable('users', {
+            id: {
+                allowNull: false,
+                autoIncrement: true,
+                primaryKey: true,
+                type: Sequelize.INTEGER,
+            },
+            email: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            password: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            username: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            nickname: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            profilePic: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            profileBanner: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            bio: {
+                allowNull: false,
+                type: Sequelize.STRING,
+            },
+            followers: {
+                allowNull: false,
+                type: Sequelize.ARRAY(Sequelize.INTEGER),
+            },
+            following: {
+                allowNull: false,
+                type: Sequelize.ARRAY(Sequelize.INTEGER),
+            },
+            posts: {
+                allowNull: false,
+                type: Sequelize.ARRAY(Sequelize.INTEGER),
+            },
+            likes: {
+                allowNull: false,
+                type: Sequelize.ARRAY(Sequelize.INTEGER),
+            },
+            shares: {
+                allowNull: false,
+                type: Sequelize.ARRAY(Sequelize.INTEGER),
+            },

```

Comment:
Okay so turns out Sequelize.ARRAY works for postgres only lol, maybe switch it to Sequelize.JSON for now for array fields